software/smederee.yml

```
@@ -0,0 +1,12 @@
+name: "Smederee"
+website_url: "https://smeder.ee"
+source_code_url: "https://smeder.ee/~jan0sch/smederee"
+description: "A frugal platform which is dedicated to help people build great software together leveraging the power of the Darcs version control system."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Scala
+tags:
+  - Software Development
+  - Software Development - Project Management
+depends_3rdparty: false
```

Fixed tags
Removed optional option. 

```suggestion
tags:
  - Software Development - Project Management
```

---

@jan0sch can you apply the changes suggested above? Thanks

---

So only one tag?

---

The software development tag is the category tag, and shouldn't be used

Just the one tag is needed here.

---

Ah okay, thanks. Changed.

---

To clarify, the problem was https://github.com/awesome-selfhosted/awesome-selfhosted-data/actions/runs/6921526064/job/18850382513: `ERROR:awesome_lint.py: 1 items tagged Software Development, each tag must have at least 3 items attached`

We split the `Software Development` category to `Software Development - SUBCATEGORY NAME` a while ago, so the top level `Software Development` does not contain projects anymore.

We could remove this tag in the future since it's no longer used and is just a placeholder: https://awesome-selfhosted.net/tags/software-development.html